txn_writer.go

```diff
@@ -36,9 +36,9 @@ const exampleTOML = `
 ## or in a dependency.
 # ignored = ["github.com/user/project/badpkg"]
 
-## Dependencies define constraints on dependent projects. They are respected by
+## Constraint define constraints on dependent projects. They are respected by
```

Comment:
This stutters a bit now, with the repetition of `constraint`. Let's modify the first sentence to read:

`Constraints are rules for how directly imported projects may be incorporated into the depgraph.`